### DISCUSSION TITLE
job id

### DISCUSSION THREAD

**Discussion by KavyaSethia** [agronholm/apscheduler#659] [Q&A]
sched.add_job(job_function,'date',timezone="Asia/Kolkata",run_date=(datetime(int(date[0:4]),int(date[4:5]),int(date[5:7]),int(date[7:9]),int(date[9:])))))
Can u tell me where i will find jod_id for this job function to remove the job

**Comment by Ben2022-cell** [comment]
sched.get_jobs() can be get the job list.
